src/Module/AdminEditServiceUiUpdateHandler.php

```diff
@@ -325,7 +325,15 @@ protected function _processSessionRuleData($serviceId, $ruleData)
             'repeat_until_date'   => strtotime($this->_containerGet($ruleData, 'repeatUntilDate')),
             'repeat_weekly_on'    => implode(',', $this->_containerGet($ruleData, 'repeatWeeklyOn')),
             'repeat_monthly_on'   => implode(',', $this->_containerGet($ruleData, 'repeatMonthlyOn')),
-            'exclude_dates'       => implode(',', $this->_containerGet($ruleData, 'excludeDates')),
         ];
+
+        $excludeDates = [];
+        foreach ($this->_containerGet($ruleData, 'excludeDates') as $_excludeDate) {
+            $excludeDates[] = strtotime(date('Y-m-d', strtotime($_excludeDate)), 0);
```

Comment:
What was the problem, how was it solved? Also, looks like this code could use some abstraction.

Reply:
Previously it was saving as date time strings. Now as timestamps, and without the time (i.e. at `00:00:00`).

Reply:
Aren't we using date-time strings now in the API? And please abstract conversion to a standard format into a method.